projects/plutus/src/x86/ltc_adc.c

```diff
@@ -0,0 +1,90 @@
+#include "ltc_adc.h"
+
+// x86 module for mocking current sense data
+
+#include "critical_section.h"
+#include "ltc2484.h"
+
+#define TEST_INPUT_VOLTAGE 1000
+
+static void prv_ltc_adc_read(SoftTimerID timer_id, void *context) {
+  LtcAdcStorage *storage = (LtcAdcStorage *)context;
+
+  storage->buffer.status = ltc2484_raw_adc_to_uv(NULL, &storage->buffer.value);
```

Comment:
There's still no point in calling the ltc2484 function in this module. Just set the voltage directly.

Reply:
Fixed